crates/control/src/motion/walking_engine.rs

```diff
@@ -378,8 +389,25 @@ impl WalkingEngine {
             right_leg: LegJoints::fill(leg_stiffness),
         };
 
-        Ok(MainOutputs {
-            walk_motor_commands: MotorCommands {
+        let walk_joints_commands = if matches!(self.walk_state, WalkState::Standing) {
+            let unoptimized_walk_joints_command = MotorCommands {
+                positions: BodyJoints {
+                    left_arm,
+                    right_arm,
+                    left_leg,
+                    right_leg,
+                },
+                stiffnesses,
+            };
+            MotorCommands {
+                positions: self.current_minimizer.optimize_body(
+                    context.sensor_data.currents,
+                    unoptimized_walk_joints_command.positions,
+                ),
+                stiffnesses,
+            }
```

Comment:
This optimization is duplicated to the one in the `motor_command_collector`, is it?

Reply:
and also has the flaw of jumping...

Reply:
Oh yes, removed the minimization in the walking engine.